### PR TITLE
Tts button

### PR DESCRIPTION
**Pull request by dmcavoy**
# Description

Makes it so you can click the whole tts button in insturctions including the volume icon instead of just clicking the play button.

## Links

- [jira](https://codedotorg.atlassian.net/browse/LP-840)

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by clareconstantine — APPROVED** [review]
Yay! I found the old behavior confusing when I first tried to use this button.

**Review by davidsbailey — APPROVED** [review]
yay bigger click targets!

File: apps/src/templates/instructions/InlineAudio.jsx [hunk 1]
```diff
@@ -245,7 +246,6 @@ class InlineAudio extends React.Component {
               this.props.style && this.props.style.button,
               this.state.hover && styles.hover
```
Comment (Hamms) [inline apps/src/templates/instructions/InlineAudio.jsx:247]: can we also move the hover logic and the appropriate styling (ie, `cursor: 'pointer'`, etc)?

Comment (dmcavoy) [reply apps/src/templates/instructions/InlineAudio.jsx:247]: I'll take a look at this. The hover seemed to be working already so maybe I should just remove the old logic for the play part.

Comment (Hamms) [reply apps/src/templates/instructions/InlineAudio.jsx:247]: I think it's just that both the play and "volume" buttons implement the same hover logic and styling, which is silly but at least kind of makes sense if they're supposed to be independent entities.

Comment (dmcavoy) [reply apps/src/templates/instructions/InlineAudio.jsx:247]: Ya it's working correctly with the hover currently so I'm going to leave it as is

**Pull request merged**